src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/smoke/PersonSmokeTest.kt

```diff
@@ -32,7 +32,7 @@ class PersonSmokeTest : DescribeSpec({
     )
 
     response.statusCode().shouldBe(HttpStatus.OK.value())
-    response.body().shouldContain("\"persons\":[")
+    response.body().shouldContain("\"content\":[")
```

Comment:
If you go with ting's suggestion to change `content` to `data`. This will need to change too

Reply:
We've decided not to change this atm and don't see a benefit to keeping it specific.